src/header/header.scss

```diff
@@ -12,6 +12,9 @@
   width: 100%;
   height: $iui-baseline * 5;
   user-select: none;
+  box-sizing: border-box;
```

Comment:
This was necessary, otherwise the `width: 100%` + padding made the header display at something like 110% screen width.  This also made the header lose a little bit in height causing the header image tests to change.

[Sanitize.scss](https://github.com/csstools/sanitize.css#box-sizing-defaults-to-border-box) suggests applying this on everything.  Maybe worth adding to all classes with `.iui-`?

Reply:
Applying it on everything will likely cause hundreds of images to change. Maybe you can combine it with the rem change, and convert this PR to draft until then.

Reply:
Looks to be 98 tests failing if we set `[class*='iui-'] { box-sizing: border-box; }`.

Reply:
I think we talked that in most of the cases we should have border-box.. that way, we can ensure out sizes are actually the ones we set in rules...
But its a huge change, again...